### ISSUE TITLE
Dysplastic Liver nodule

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11697]
Label: Dysplastic Liver Nodule
Def: A precancerous lesion in which hepatocytes show morphologic alterations and, depending on the degree of atypia. 
MedGen UID: 
272706
 
Synonym: Dysplastic hepatic nodule, dysplastic nodule, Heteroproliferative nodules

Parent term: HP:0002896 Neoplasm of the liver (although this is pre-neoplasm?)

Possible child terms: 
Label: Low grade dysplastic nodule, LGDN
Def: Mild cellular heterogeneity, often without a distinct nodular structure.
Label: High grade dysplastic nodule: More pronounced cellular structural heterogeneity
https://doi.org/10.1007/978-981-97-2985-2_15

**Comment by pnrobinson** [comment]
added
Adding new term: 
- ID: HP:6001545
- label: Dysplastic Liver Nodule
- parents: [Neoplasm of the liver (HP:0002896)](https://hpo.jax.org/app/browse/term/HP:0002896)
- pmid: 
- def: A precancerous lesion in which hepatocytes show morphologic alterations and, depending on the degree of atypia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11697
- synonyms: Dysplastic hepatic nodule(NONE)
